doc/source/contributing/quest-available-datasets.rst

```diff
@@ -22,4 +26,4 @@ Want to add a new dataset to QUEST? No problem! QUEST includes a template script
 
 Guidelines on how to construct your dataset module may be found here: (link to be added)
```

Comment:
link still missing!

Reply:
I was looking around, and I can't find a document/page on making a dataset module. Should we reword this to refer to the template script and "How to Contribute" page?

Reply:
I think we were planning to make a notebook/page to this effect but haven't done so... this may be a good candidate for our "next steps" list.

Reply:
Got it. I'll rearrange the text a bit, then mention that detailed guidelines are a work in progress.